index.html

```diff
@@ -0,0 +1,203 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Online COnference Website</title>
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v6.1.1/css/all.css" />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200"
+    />
+    <link rel="stylesheet" href="css/styles.css" />
+</head>
+<body>
+    <div class="sns">
+        <ul>
+            <li>
+                <a href="#"><i class="fab fa-facebook-f"></i></a>
+            </li>
+            <li>
+                <a href="#"><i class="fab fa-twitter"></i></a>
+            </li>
+            <li>
+                <a href="">English</a>
+            </li>
+            <li>
+                <a href="#">My Page</a>
+            </li>
+            <li>
+                <a href="#">Logout</a>
+            </li>
+        </ul>
+    </div>
+    <!-- Header section -->
+    <header class="header-bg">
+        <span><a href="index.html"><span class="cyber">Cyber<strong class="world">World</strong></span></a></span>
+        <nav id="nav-bar">
+            <ul class="nav-menu">
+                <li class="nav-item"><a href="about.html" class="nav-link">About</a></li>
+                <li class="nav-item"><a href="#" >Program</a></li>
+                <li class="nav-item"><a href="#" >Join</a></li>
+                <li class="nav-item"><a href="#" >Sponsor</a></li>
+                <li class="nav-item"><a href="#" >News</a></li>
+                <li class="nav-item"><a href="#"><span class="tour">CyberWorld</span></a></li>
+            </ul>
+            <a href="#nav-bar" class="hamburger" aria-label="Open main menu"><i class="fas fa-bars" aria-hiiden="true"></i></a>
+        </nav>

```

Comment:
- [ ] When I click on the hamburger button in the mobile view the mobile menu doesn't appear on the page as expected. and the same happens on the about page.
So please fix that on both pages.

![chrome-capture-2022-5-23](https://user-images.githubusercontent.com/68796232/175381974-962cc9c6-9f21-438b-8fe5-6d5ebd0dff13.gif)

Reply:
I have implemented the mobile menu as requested